Formula/root.rb

```diff
@@ -62,7 +52,11 @@ def install
               "http://lcgpackages",
               "https://lcgpackages"
 
-    args = std_cmake_args + %W[
+    if MacOS.version < :big_sur
+      ENV.append "CXXFLAGS", "-isystem/Library/Developer/CommandLineTools/usr/include/c++/v1"
+    end
+
+    args std_cmake_args + %W[
```

Comment:
```suggestion
    args = std_cmake_args + %W[
```

Reply:
That shouldn't have passed the linter I'd think 🤦